preupg/creator/ui_helper.py

```diff
@@ -128,7 +126,35 @@ def specify_upgrade_path(self):
         return True
 
     def prepare_content_env(self):
-        self.content_path = os.path.join(self.get_group_name(), self.get_content_name())
+        self.content_path = os.path.join(self.get_group_name(),
+                                         self.get_content_name())
+
+    def properties_ini(self):
```

Comment:
The function name should convey its purpose and should contain a verb (what it does). If it's hard to come up with a good function name then the function is not properly written or decomposed. I would:
- remove this function
- set the `self.properties_ini_path` in the __init__ of UIHelper
- move the `if not self.properties_ini_exists():` check to the `get_properties_ini_versions()` itself